### PR TITLE
Update gh spec

### PR DESCRIPTION
**Pull request by EliasGcf**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Update the `gh` (GitHub's CLI tool) spec.

### CODE REVIEW

File: dev/gh.ts [hunk 1]
```diff
@@ -14,43 +14,666 @@ const ghGenerators: Record<string, Fig.Generator> = {
...
+    postProcess: (out) => {
+      const aliases = out.split("\n").map((line) => {
+        const [name, content] = line.split(":");
+
+        return { name: name.trim(), content: content.trim() };
+      });
+
+      return aliases.map(({ name, content }) => ({
+        name,
+        displayName: name,
```
Comment (QuiiBz) [inline dev/gh.ts:28]: There is no need to set `displayName `if it's the same as `name`

File: dev/gh.ts [hunk 2]
```diff
@@ -14,43 +14,666 @@ const ghGenerators: Record<string, Fig.Generator> = {
...
+                "The protocol to use for git clone and push operations",
+              args: {
+                name: "option",
+                suggestions: ["https", "ssh"],
+              },
+            },
+            {
+              name: "editor",
+              description: "The text editor program to use for authoring text",
+              args: { name: "editor" },
```
Comment (QuiiBz) [inline dev/gh.ts:152]: Could we suggest `vim`, `nano`, ... here?

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this amazing PR!

**Comment by EliasGcf** [comment]
Hi, @QuiiBz, I've already made the changes.

**Pull request merged**